src/main/java/heat/utils/DTOToDBConverter.java

```diff
@@ -0,0 +1,24 @@
+package heat.utils;
+
+import heat.db.User;
+import heat.db.UserParameters;
+import heat.dto.UserSettings;
+
+public class DTOToDBConverter {
+    public static User convertUser(UserSettings userSettings) {
```

Comment:
`convertUser` sounds like we want to convert `User` to smth. Mb `convertToUser`?

Reply:
Done.